tck/src/test/java/org/reactivestreams/tck/IdentityProcessorVerificationTest.java

```diff
@@ -115,6 +126,155 @@ public void required_spec104_mustCallOnErrorOnAllItsSubscribersIfItEncountersANo
+              try {
+                subscriber.onNext(elem);
+              } catch (Exception ex) {
+                env.flop(ex, String.format("Calling onNext on [%s] should not throw! See https://github.com/reactive-streams/reactive-streams-jvm#2.13", subscriber));
+              }
+            }
+          }
+
+          @Override
+          public void onError(Throwable t) {
+            for (Subscriber<? super Integer> subscriber : subscribers) {
+              try {
+                subscriber.onError(t);
+              } catch (Exception ex) {
+                env.flop(ex, String.format("Calling onError on [%s] should not throw! See https://github.com/reactive-streams/reactive-streams-jvm#2.13", subscriber));
+              }
+            }
+          }
+
+          @Override
+          public void onComplete() {
+            for (Subscriber<? super Integer> subscriber : subscribers) {
+              try {
+                subscriber.onComplete();
+              } catch (Exception ex) {
+                env.flop(ex, String.format("Calling onComplete on [%s] should not throw! See https://github.com/reactive-streams/reactive-streams-jvm#2.13", subscriber));
+              }
+            }
+          }
+
+          private Subscription createSubscription(CountDownLatch awaitLatch, final Subscriber<? super Integer> s, final AtomicLong demand) {
+            final MySubscription sub = new MySubscription(awaitLatch, s, demand);
+            subs.add(sub);
+            subscribers.add(s);
+            return sub;
+          }
+
+          final class MySubscription implements Subscription {
+            private final CountDownLatch awaitLatch;
+            private final Subscriber<? super Integer> s;
+            private final AtomicLong demand;
+
+            public MySubscription(CountDownLatch awaitTwoLatch, Subscriber<? super Integer> s, AtomicLong demand) {
+              this.awaitLatch = awaitTwoLatch;
+              this.s = s;
+              this.demand = demand;
+            }
+
+            @Override
+            public void request(final long n) {
+              ex.execute(new Runnable() {
+                @Override
+                public void run() {
+                  demand.addAndGet(n); // naive, but good enough for the test
+                  awaitLatch.countDown();
+                  try {
+                    awaitLatch.await(env.defaultTimeoutMillis(), TimeUnit.MILLISECONDS);
+                    while (demand.getAndDecrement() > 0) {
+                      upstreamSubscription.request(1);
+                    }
+                  } catch (InterruptedException e) {
+                    env.flop(e, "Interrupted while awaiting for all downstreams to signal some demand.");
+                  }
+
+                }
+              });
+            }
+
+            @Override
+            public void cancel() {
+              demand.set(Long.MIN_VALUE); // naive but OK for this test
```

Comment:
It's naive "let's act as if it's cancelled", since we won't signal upstream requests if `demand < 0`.
It's simple enough for this case, as it only needs to run a bit in order for the errors to be signalled in the meantime.

I made it more explicit now